src/main/java/racingcar/view/input/Input.java

```diff
@@ -0,0 +1,65 @@
+package racingcar.view.input;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import racingcar.view.input.error.InputError;
+import racingcar.view.input.error.InputIllegalArgumentException;
+
+public final class Input {
+
+    public static final Pattern REMOVE_REGEX_PATTERN = Pattern.compile("\\s");
+    public static final String DIVIDING_STANDARD = ",";
+    public static final int MAXIMUM_NAME_LENGTH = 5;
+    public static final int MINIMUM_NAME_LENGTH = 1;
+
+    public static List<String> inputNames() {
+        String name = Console.readLine();
+
+        return devideToList(name);
+    }
+
+    public static int inputTrialAmount() {
+        String trialAmount = Console.readLine();
+        validateNotBlank(trialAmount);
+
+        try {
+            return Integer.parseInt(trialAmount);
+
+        } catch (InputIllegalArgumentException e) {
+            throw new InputIllegalArgumentException(InputError.MUST_BE_NUMBER);
+        }
+    }
+
+    private static List<String> devideToList(String value) {
+        String removedSpaceValue = removeSpace(value);
+        List<String> dividedName = Arrays.stream(removedSpaceValue.split(DIVIDING_STANDARD)).toList();
+
+        validateNameLength(dividedName);
+
+        return dividedName;
+    }
+
+    private static String removeSpace(String value) {
+
+        return REMOVE_REGEX_PATTERN.matcher(value).replaceAll("");
+    }
+
+    private static void validateNameLength(List<String> names) {
+        long countNames = names.stream()
+                .dropWhile(name -> name.length() >= MINIMUM_NAME_LENGTH && name.length() <= MAXIMUM_NAME_LENGTH)
```

Comment:
```suggestion
.dropWhile(name -> isNameValideLength(name))
```
이 부분이 길어서 가독성을 고려해, 메소드를 분리해보는 건 어떨까요?
```java
    private static boolean isNameValideLength(String name) {
        return name.length() >= MINIMUM_NAME_LENGTH && name.length() <= MAXIMUM_NAME_LENGTH;
    }
```

Reply:
dropWhile 사용법..! 배워갑니다. 저 역시 @skylar1220 님처럼 가독성을 고려하면 좋을 것 같아요!

Reply:
비교 구문이라서 조금 고민이 되었던 부분인데
역시 method로 분리하는 것이 보기 좋네요!
현지님 늘 좋은 의견 감사합니다! 🥰🥰